tests/integration/test_able_to_sent_to_commands.py

```diff
@@ -0,0 +1,190 @@
+import time
+import json
+import random
+
+import pytest
+
+import paho.mqtt.client as mqtt
+from pymodbus.client import ModbusTcpClient
+from pymodbus.payload import BinaryPayloadDecoder
+from pymodbus.constants import Endian
+
+def test_able_to_update_coil():
+    random_value = random.randint(1, 2) % 2 == 0
+
+    message_dictionary = {
+        "action": "testRegister",
```

Comment:
The test is about updating coils, and reads back data using `read_coils` with an address configured for `testCoil` ... but the message action is `testRegister` 🤔  should this work?

Reply:
Ideally, I'd argue, it should fail.